### ISSUE TITLE
Problems with experimental data in the paper

### ISSUE THREAD

**Issue by liujuncn** [thuml/iTransformer#41]
I ran the script from the official PatchTST repository, and the metrics I obtained are better than the results in your paper.

https://github.com/yuqinie98/PatchTST

For example, the results for the weather dataset are:
![image](https://github.com/thuml/iTransformer/assets/902005/3f9e4dbe-5cf4-4e3b-a11d-aaf9e23e1995)

your results are:
![image](https://github.com/thuml/iTransformer/assets/902005/6c3f76be-5b33-4fb1-a9dc-94faa340a2c0)

I am wondering if there are any differences between the implementation of your experimental code and theirs?

**Comment by WenWeiTHU** [comment]
Hello, thanks for your concen. There's a lot of difference here:

* Enlarged lookback window: PatchTST paper adopts tunable lookback length in {336, 512}, while ours adopts the unified length of 96, following the unified long-term forecasting protocol of [TimesNet](https://github.com/thuml/Time-Series-Library).
* More epochs to train: PatchTST paper trains the PatchTST model with 100 epochs, while we train all models with 10 epochs.
* Learning rate: PatchTST paper adapts a carefully designed learning rate strategy.

> Related issue: https://github.com/thuml/iTransformer/issues/30

**Comment by liujuncn** [comment]
I would like to test a new backbone architecture that replaces the Transformer in your code experiment setup. What are the things I should be aware of? 

Thanks in advance!

**Comment by WenWeiTHU** [comment]
Hello, you can refer to this [repo](https://github.com/thuml/Time-Series-Library), since iTransformer in built on this!
